waterlog/src/actions/segmentLeaksActions.js

```diff
@@ -0,0 +1,13 @@
+import { FETCH_SEGMENTS_LEAKS } from "./types";
+
+export const fetchSegmentsLeaks = () => dispatch => {
+  console.log("please");
```

Comment:
Please do not leave console logs in the code base. Overtime what happens is that the console gets cluttered with random logs and it becomes useless. If you need to debug Redux, have a look at this article https://code.visualstudio.com/docs/nodejs/reactjs-tutorial#_debugging-react for getting it setup